### ISSUE TITLE
Sachin Kohare iOS assignment request

### ISSUE THREAD

**Issue by skohare-tech** [nets-dev/assignment#30]
Hi, 
     My name is Sachin Kohare. I have about 3+ years of experience working as an iOS developer and have experience in Swift. You may grant my GitHub account access to the iOS code assignment. I am free this week and will start working on the project as soon as I get the access. 

Thanks!

**Comment by nets-dev** [comment]
Hi @skohare-tech, please use your personal GitHub account with some projects for reference.
As a developer, projects speak louder than years of experience or CV.
(Any project in iOS, even tutorial/school projects would be appreciated)

**Comment by skohare-tech** [comment]
Hello @nets-dev  
I have added the some practice project in the same account

**Comment by nets-dev** [comment]
@skohare-tech Is there even a single Swift file in this practice project you've uploaded?

**Comment by skohare-tech** [comment]
@nets-dev   That was in another repo in same account. Now I have added into the assignment repo also.

**Comment by nets-dev** [comment]
@skohare-tech The point is, there is no Swift code in the repository you've provided. 
This is an iOS position and we expect applicants to have personal project/s with Swift or Objective C code. 
Do you have any project with Swift or Objective C code that you've written?

**Comment by skohare-tech** [comment]
@nets-dev  Yes i have added in your shared in your shared assignment repo and I am able to see also.
And I have added into my own repo also, You can check on https://github.com/skohare-tech/PracticeDemoCode this location.

**Comment by nets-dev** [comment]
@skohare-tech Can you provide a direct link to a single Swift file in this set of projects? 
The last file extension to your link should be `.swift`.  
e.g. `https://github.com/skohare-tech/PracticeDemoCode/SomeFolder/SomeSwiftFile.swift`

**Comment by skohare-tech** [comment]
Hello @nets-dev  
I have checked, You are right there swift file is not uploaded, I am checking for that what is the problem and will update you.

**Comment by nets-dev** [comment]
@skohare-tech Forked projects are not your own. Try to upload code that you've worked on at school, in tutorial or your personal project!

**Comment by nets-dev** [comment]
@skohare-tech That will not be necessary. Ability to use `git` is one of our basic requirements so we advice you to build your skills by working on personal projects and uploading to GitHub in the future. Thank you! 

(We'll still grant you access to the project if you manage to upload a project to GitHub in the next few days)

**Comment by skohare-tech** [comment]
@nets-dev  I am agree with you. It's basic requirement and I am also working with BitBucket since 3 years in team. I have good knowledge about version control. Since 3 days I was busy that's why not given time on that.
I have added code and checked also, now and you can check by url https://github.com/skohare-tech/PracticeProjects
Sorry for the delaying.

**Comment by nets-dev** [comment]
@skohare-tech This is still not satisfying however we have invited you to the project. Can you confirm that you have access?

**Comment by skohare-tech** [comment]
Hi @nets-dev,
I have received your request and checked task also. The thing is that you requested for the SwiftUI development but I don't have experience with the SwiftUI, So I have request to accept the development with storyboard UI.

**Comment by nets-dev** [comment]
@skohare-tech Learning is a big part of software development. The UI is quite simple and we appreciate what you're able to learn and implement if you decide to take the challenge. Best of luck!
